source/extensions/filters/udp/udp_proxy/config.h

```diff
@@ -174,6 +180,8 @@ class UdpProxyFilterConfigImpl : public UdpProxyFilterConfig,
   const std::chrono::milliseconds session_timeout_;
   const bool use_original_src_ip_;
   const bool use_per_packet_load_balancing_;
+  bool flush_access_log_on_tunnel_connected_;
```

Comment:
nit: ``const bool``

Reply:
It can't be const (initialized in constructor and not in the initializer list)